package.json

```diff
@@ -14,11 +14,11 @@
     "ajv": "8.17.1",
     "ajv-formats": "3.0.1",
     "axios": "1.7.8",
+    "dayjs": "1.11.3",
```

Comment:
I didn't switch to the latest version of `dayjs` to minimize the diffs in the `package-lock.json`.

Reply:
I'm sure we'll get a dependabot PR for that, which will be nice to keep it isolated.